README.md

```diff
@@ -157,13 +159,10 @@ The legend is shown below in the table format
 
 To start mutation test, run the following command from the repo directory:
 ```bash
-$ move-mutation-test run --package-dir move-mutator/tests/move-assets/simple -o report.txt
+$ move-mutation-test run --package-dir move-mutator/tests/move-assets/simple --output report.txt
```

Comment:
Let's move this one one up, or have two sections for the readme, and have the `move-mutation-test` install and usage show up separately and at the top.